docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md

```diff
@@ -0,0 +1,14 @@
+Nebula use `DOWNLOAD` command to get `SST` files from `HDFS`.
```

Comment:
Nebula downloads `SST` files into the download directory with `DOWNLOAD` command.